source/vibe/core/log.d

```diff
@@ -594,6 +598,9 @@ final class SyslogLogger : Logger {
 		assert(lines[5] == "<138>1 0000-01-01T00:00:00.000001 - " ~ BOM ~ "appname - - - " ~ BOM ~ "αβγ\n");
 		assert(lines[6] == "<137>1 0000-01-01T00:00:00.000001 - " ~ BOM ~ "appname - - - " ~ BOM ~ "αβγ\n");
 		removeFile(fstream.path().toNativeString());
+		
+		version(VibeLibasyncDriver)
+			destroyAsyncThreads();
```

Comment:
Ah yes, that's because the logger uses a file stream which requires the thread pool on this driver :-p

Thinking about the logger, I had to do some adjustments to remove GC allocations so that I could use it in destructors: https://github.com/etcimon/vibe.d/blob/master/source/vibe/core/log.d#L123